.github/compatibility-test-matrices/release-v5.0.x.json

```diff
@@ -1,12 +1,485 @@
 {
   "include": [
+    {
+      "chain-binary": "simd",
+      "chain-a-tag": "release-v5.0.x",
```

Comment:
Food for thought: Base tests which should work for any version combination, maybe it'd be possible to swap out the "branch-to-test" and then only maintain files for tests which require logic for which cross versions to test

Reply:
Actually I was just speaking with @crodriguezvega about this and I think we may be able to make some changes and leverage some of the built in matrix generation features of github actions to make some of these permutations re-usable (swap out the branch as you said).

I will update the versions as you suggested and take a look into seeing if this can be simplified to work a bit better with our use case!